### ISSUE TITLE
Guild tensorboard to accept tensorboard args

### ISSUE THREAD

**Issue by FelixAbrahamsson** [guildai/guildai#144]
Would be nice to be able to pass tensorboard flags like `--samples_per_plugin` when running `guild tensorboard`.

**Comment by christabella** [comment]
Agreed, being able to do `--samples_per_plugin images=100` would be handy since the default is very low (10)

**Comment by gar1t** [comment]
0.7 will support a `-O` option that can be used to pass TB options through. Like this:

```
$ guild run tensorboard -O samples_per_plugin=images=100
```

You can verify the options passed to TB by including the `--debug` option for the `guild` command:

```
guild --debug tensorboard ...
```

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report!
